tests/airport.spec.js

```diff
@@ -0,0 +1,450 @@
+const { assertEquals } = require('../test-framework');
+const Airport = require(`../src/Airport`);
+const Plane = require(`../src/Plane`);
+const Weather = require('../src/Weather');
+
+let expected, actual, result, airport, plane;
+
+// Test 1 - planesLanded length increases to 1 when an empty airport is told to land a plane
+console.log(`==========`);
+console.log(`planesLanded length increases to 1 when an airport is told to land a plane`);
+
+// Arrange
+airport = new Airport();
```

Comment:
Later tests required an Airport constructor, and this part should probably be changed to accommodate that (I think I accounted for the change in the constructor instead)